### ISSUE TITLE
Icons from new Fluent design

### ISSUE THREAD

**Issue by PedroGuilhermeFariaDuarte** [react-icons/react-icons#451]
Hello!

Windows 11 have a new UI and new ICONS :)

Please, added the news icons, follow  most popular [Fluent Icons Package](https://www.npmjs.com/package/@fluentui/react-icons)

**Comment by yashash1511** [comment]
yes please [fluentui icons](https://github.com/microsoft/fluentui-system-icons) are one of the best

**Comment by hu-tu-yi-shi** [comment]
yes, its a good icons,  here is more info https://react.fluentui.dev/?path=/docs/concepts-developer-icons-icons-catalog--page

**Comment by matiascaputti** [comment]
any updates on this?
